src/nativeRenderManager.js

```diff
@@ -48,25 +48,25 @@ export function newNativeRenderManager(win) {
   }
 
   // START OF MAIN CODE
-  let renderNativeAd = function(nativeTag) {
+  let renderNativeAd = function(doc, nativeTag) {
```

Comment:
While running a regression test on the using the `native-render.js` type creative setup, I encountered an undefined error when the creative started to call the `pbNativeTag.renderNativeAd()` via the new `legacyNativeRender.js` file.

The issue is in the call signature for this function here.  Previously there was a single argument for the `nativeTag` which contained the pubUrl and other info.  Now with the updated signature, this data getting passed into the `doc` property instead of the `nativeTag` 2nd param, which breaks the rest of the logic in this function.

We need some new logic here to handle the old scenario since this file is shared between the old/new native rendering routes.

Reply:
I changed file `legacyNativeRender` so the `renderNativeAd` function that is exposed accepts only one parameter. I also added two tests for it.

Reply:
Hi @musikele 

When I tried to test this out again on a test page, I was still seeing the behavior.  I attached a screenshot for reference.
I saw it attempt to use the new call function you added, but I think the call interpreted the `document` variable as the `this` context for the call function rather an argument to pass through to the subsequent function.  Can you take a look again?


<img width="1904" alt="Screen Shot 2022-09-27 at 10 08 02 AM" src="https://user-images.githubusercontent.com/31102355/192549536-e181a7f8-e8c2-4c38-8025-3a9a307b98d3.png">

Reply:
Confirmed fix per the latest updates.